spec/toy_robot_spec.rb

```diff
@@ -31,48 +46,68 @@
     specify 'When y is more than four, it expect to raise an exception' do
       expect { subject.place(0,5,:north) }.to raise_error(InvalidPositionError, "Invalid Position, 0, 5, north")
     end
-    specify 'When valid it should move Robot to position and update facing' do
+    specify 'When given valid x and y, it should move Robot to position and update facing' do
       subject.place(3,4,:west)
       expect(subject.x).to eq(3)
       expect(subject.y).to eq(4)
       expect(subject.facing).to eq(:west)
     end
 
+    specify 'When given valid x and y, the array @movements should have only 1 item' do
+      subject.place(1,2,:north)
+      expect(subject.movements.length).to eq(1)
+    end
+
+    specify 'When given valid x and y, the 1st item in array should have the place coordinate and facing' do
+      subject.place(1,2,:north)
+      expect(subject.movements[0]).to eq("PLACE 1, 2, NORTH")
+    end
   end ## end of describe 'place'
 
   describe 'move' do
-    subject { ToyRobot.new }
     context 'When the Robot is at Origin (0, 0, :north)' do
-      it 'it will move one position forward' do
-        subject.move
-        expect(subject.x).to eq(0)
-        expect(subject.y).to eq(1)
-        expect(subject.facing).to eq(:north)
+      it 'will move one position forward' do
+        robot = ToyRobot.new
```

Comment:
use the explicit place command as below. You are testing the #move command, not the initialiser here so be explicit. You are binding expected move behaviour to the initialiser. You should be able to change the initialiser to face a different way without breaking this test. There is no benefit to not following the pattern demonstrated below (unless i'm missing something).